### PR TITLE
Docstring for Markdown.html and Markdown.latex

### PR DESCRIPTION
**Pull request by fonsp**
Part of #52725

Feel free to edit/close/merge without comment

### CODE REVIEW

File: stdlib/Markdown/src/render/html.jl [hunk 1]
```diff
@@ -182,6 +182,17 @@ htmlinline(io::IO, x) = tohtml(io, x)
 
 export html
 
+"""
+    html([io::IO], md)
+
+Render a Markdown object to its HTML representation. If no `io` is provided, the result will be returned as a `String`.
```
Comment (stevengj) [inline stdlib/Markdown/src/render/html.jl:188]:
```suggestion
Output the contents of the Markdown object `md` in HTML format, either
writing to an (optional) `io` stream or returning a string.

One can alternatively use `show(io, "text/html", md)` or `repr("text/html", md)`, which
differ in that they wrap the output in a `<div class="markdown"> ... </div>` element.
```

File: stdlib/Markdown/src/render/latex.jl [hunk 2]
```diff
@@ -167,6 +167,16 @@ function latexesc(io, s::AbstractString)
     end
 end
 
+"""
+    latex([io::IO], md)
+Convert a Markdown object to a LaTeX expression. If no `io` is provided, the result will be returned as a `String`.
```
Comment (stevengj) [inline stdlib/Markdown/src/render/latex.jl:172]:
```suggestion
```suggestion
show(io::IO, ::MIME"text/latex", md::MD) = latex(io, md)

"""
    latex([io::IO], md)

Output the contents of the Markdown object `md` in LaTeX format, either
writing to an (optional) `io` stream or returning a string.

One can alternatively use `show(io, "text/latex", md)` or `repr("text/latex", md)`.
``````

Comment (stevengj) [inline stdlib/Markdown/src/render/latex.jl:172]:
```suggestion
"""
    latex([io::IO], md)

Output the contents of the Markdown object `md` in LaTeX format, either
writing to an (optional) `io` stream or returning a string.

One can alternatively use `show(io, "text/latex", md)` or `repr("text/latex", md)`.
```

**Comment by stevengj** [comment]
(I have a feeling that these should be deprecated in favor of `show` and `repr`, but that's a conversation for another PR.)

**Comment by fonsp** [comment]
Thanks! Lets merge?

**Review by stevengj — APPROVED** [review]

**Comment by stevengj** [comment]
Note that #52723 should be updated once this is merged (assuming it is merged first).

File: stdlib/Markdown/src/render/html.jl [hunk 3]
```diff
@@ -182,6 +182,21 @@ htmlinline(io::IO, x) = tohtml(io, x)
...
+Output the contents of the Markdown object `md` in HTML format, either
+writing to an (optional) `io` stream or returning a string.
+
+One can alternatively use `show(io, "text/html", md)` or `repr("text/html", md)`, which
+differ in that they wrap the output in a `<div class="markdown"> ... </div>` element.
+
+# Example
+```jldoctest
+julia> html(md"hello _world_")
+"<p>hello <em>world</em></p>\n"
```
Comment (stevengj) [inline stdlib/Markdown/src/render/html.jl:197]:
```suggestion
"<p>hello <em>world</em></p>\\n"
```

File: stdlib/Markdown/src/render/latex.jl [hunk 4]
```diff
@@ -167,6 +167,20 @@ function latexesc(io, s::AbstractString)
...
+
+Output the contents of the Markdown object `md` in LaTeX format, either
+writing to an (optional) `io` stream or returning a string.
+
+One can alternatively use `show(io, "text/latex", md)` or `repr("text/latex", md)`.
+
+# Example
+```jldoctest
+julia> latex(md"hello _world_")
+"hello \\emph{world}\n\n"
```
Comment (stevengj) [inline stdlib/Markdown/src/render/latex.jl:181]:
```suggestion
"hello \\\\emph{world}\\n\\n"
```

**Comment by mitiemann** [comment]
After merging, please see whether #52606 now builds successfully and merge, if appropriate.

**Pull request merged**